### PR TITLE
Adjust header size to searchbarsize, Logo visible in full height

### LINKED ISSUE #59: Header overlaps images
**Describe the bug**
On the page Home the header overlaps the images of the carousel.

**To Reproduce**
1. Go to 'Home'
2. Open the inspector view of your browser (F12 in Firfox)
3. Use the element selection tool
4. Select the image carousel

**Expected behavior**
The image carousel should start below the header.

**Screenshots**
grafik

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser: Firefox
 - Version 77

**Smartphone (please complete the following information):**
Not used for testing.

**Additional context**
None

### PR DESCRIPTION
**Pull request by andresesam**
Pretty small, no review needed i guess.

Close #59

### CODE REVIEW

**Comment by jschurse** [comment]
On mobile there should be a bit of padding:
grafik

File: app/src/components/layout/Toolbar.vue [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <template>
-    <v-app-bar flat color="#00254f">
+    <v-app-bar flat height="100vh" color="#00254f">
```
Comment (jschurse) [inline app/src/components/layout/Toolbar.vue:2]: 100vh would be the total visible vertical space @see https://www.w3schools.com/cssref/css_units.asp
I think the v-app-bar interprets this as a px value. Removing the vh from the 100 does not change the behaviour (even 100bananas has the same result).
Please remove the unit or use the correct one.

**Comment by andresesam** [comment]
> On mobile there should be a bit of padding:
> grafik

Hey Jenja, yeah i know. The bar is not responsive and i tought this is covered here #61

**Comment by jschurse** [comment]
Okay. Then it needs to be done there.

**Comment by andresesam** [comment]
> > On mobile there should be a bit of padding:
> > grafik
> 
> Hey Jenja, yeah i know. The bar is not responsive and i tought this is covered here #61 The issue itself was only for the overlapping image and the size of the bar

File: app/src/components/layout/Toolbar.vue (hunk 1 above)
Comment (andresesam) [reply app/src/components/layout/Toolbar.vue]: You're absolutely right 😄 I checked out the older version which contained the vh unit. Just changed it to px

**Comment by FiedlerJan** [comment]
> Okay. Then it needs to be done there.

Yes, this should be handled by issue #61. There are even more problems with the responsiveness of the homepage which should all be addressed together.

**Pull request merged**